viewShip.php

```diff
@@ -0,0 +1,52 @@
+<?php
+require __DIR__.'/bootstrap.php';
+// page to display a single ship data with the options to:
+// ADD, EDIT, DELETE
+use Service\Container;
+
+
+
+$id = $_GET['id'];
+
+// Need shiploader obj for ship
+$container = new Container($configuration);
+$shipLoader = $container->getShipLoader();
+$ship = $shipLoader->findOneById($id); // values may be getting changed here
+
+require 'layout/header.php';
+
+?>
+
+    <div class="container">
+            <div class="page-header">
```

Comment:
I think a breadcrumb would be perfect for this. 

Something like: `Manage Ships > [shipName] Profile` and 'Manage Ships' is a link back to the manage page.

Then you can add onto this for the edit in the future: `Manage Ships > [shipName] Profile > Edit`

Reply:
Put in a dummy breadcrumb trail. Working on implementation for a fully fledged one.